anonymizer/parse.py

```diff
@@ -9,10 +10,10 @@
 lang = "es"
 # lang = "fr"
 # path where the input files can be found
-path_input = "./data/in/"
+path_input = Path("./data/in/")
```

Comment:
here then you can continue passing strings. It is also ok if execution of `parse.py` does not work anymore, it will change completely and does not need to be adapted at this point.
I suggest importing the package and working on single functions for the moment, either through the Python console, or through Jupyter notebooks, or using pytest (learning about pytest will be the next step).